Dockerfile

```diff
@@ -1,25 +1,23 @@
-FROM python:3-stretch
+FROM python:3.6
```

Comment:
Could we get rid completely of this file in favour of `devops/docker/tasking-manager/Dockerfile`?

Reply:
oh, dropping this!

Reply:
Perhaps we can also delete `requirements.aws.txt`, `.ebignore` and `.ebextensions` within the same clean up commit. What do you think?

Reply:
yep!